src/app/delegation_backend/src/sheets.go

```diff
@@ -0,0 +1,34 @@
+package delegation_backend
```

Comment:
It would be interesting to have some code documentation briefly describing what this service does inside delegation backend.